core/trino-main/src/main/java/io/trino/security/AccessControlManager.java

```diff
@@ -1442,6 +1397,32 @@ public Map<ColumnSchema, ViewExpression> getColumnMasks(SecurityContext context,
         }
     }
 
+    @Override
+    public void checkCanSetEntityAuthorization(SecurityContext securityContext, String ownedKind, List<String> name, TrinoPrincipal principal)
```

Comment:
Can we use `io.trino.metadata.QualifiedObjectName` instead of `List<String>`?

Reply:
No, we can't use `QualifiedObjectName` because it is based on the assumption that all ownable entity kinds are contained in a catalog and schema.

Reply:
See also `io.trino.spi.connector.EntityKindAndName`, which is a pair of `String entityKind` and `List<String> name`. BTW, perhaps we could use it here as well.

Reply:
This is a reasonable suggestion, and is consistent with `checkCanGrantEntityPrivilege`.

I made this change everywhere except `denySetEntityAuthorization`.  I didn't make it there because there are dozens of calls